docs/usage/DirectMigrationExampleAPI.md

```diff
@@ -0,0 +1,200 @@
+#Direct Migration using API
+
+Please setup, deploy and configure the Migration Operator as well as the Migration Controller
+appropriately before proceeding further.
+
+We will be performing a migration of a Django Application consisting of persistent volumes and image streams, this
+application is already present on the source cluster used in the following migration example.
+
+The steps to perform a Direct Migration using Migration Toolkit for Containers (MTC) are as follows:
+
+**1. Add Migration clusters**<br>
+This step involves addition and configuration of the source as well as the destination clusters 
+involved in the migration.
+
+At first, let's create a MigCluster instance in order to add the destination cluster. 
+In our case this cluster is also the host cluster where Migration controller resides and is
+already configured, you do not need to add this cluster. 
+
+Store the destination MigCluster yaml manifest in `destination-migcluster.yaml`:
+```
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigCluster
+metadata:
+  name: host
+  namespace: openshift-migration
+spec:
+  isHostCluster: true
+```
+
+Command to create destination MigCluster instance:
+``` 
+oc create -f destination-migcluster.yaml
+```
+
+Now let's add the second cluster, that is the source cluster.
+
+Store the source MigCluster yaml manifest in `source-migcluster.yaml`:
+``` 
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigCluster
+metadata:
+  name: src-ocp-3-cluster
+  namespace: openshift-migration
+spec:
+  exposedRegistryPath: docker-registry-default.apps.mycluster-ocp3.mg.dog8code.com
+  insecure: true
+  isHostCluster: false
+  serviceAccountSecretRef:
+    name: src-ocp-3-cluster-k2vnk
```

Comment:
Where is this SA Secret ref name coming from?

Reply:
Good question worth calling out; if you are building a migration from scratch, the first thing you'd need to do is create a Secret _somewhere_ (that confirms to our expected structure with the correct fields) and give it a name of your choice. Then you set this reference details with the name/namespace to point to the secret you created with your details.

Reply:
@eriknelson Is there an example somewhere of this secret? I recall documenting something similar for [RHV VM import](https://docs.openshift.com/container-platform/4.6/virt/virtual_machines/importing_vms/virt-importing-rhv-vm.html#virt-importing-vm-cli_virt-importing-rhv-vm) to CNV from the CLI. I assume that some of the fields will be different, like `ovirt`.

Reply:
@djwhatle Any chance you know if this is documented somewhere in the controller?

We definitely need to get the structure of what this secret has to look like; it has to be exactly correct or some unexpected errors will throw.